### ISSUE TITLE
Obsolete 'reproduction of symbiont in host'? 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#20203]
Hello, 

'GO:0061887 reproduction of symbiont in host' has only been used for one paper PMID:25158041 by CACAO. In this paper, the authors look at "intramacrophage growth phenotype in mutants of the respective genes that were deficient for intramacrophage growth". The mechanisms underlying the ability to reproduce are not characterized further. 

I suggest we obsolete this term. 'reproduction of symbiont in host' is a property of the specific host-symbiont interaction, it's not a specific process; the symbiont just reproduces in that specific environment. 

@dsiegele @mgiglio99 @ValWood @nsuvarnaiari 

Does that seem ok with you ? 

Thanks, Pascale

**Comment by mgiglio99** [comment]
Hi Pascale - I see what you mean and I agree this is a problematic term. The only time I see needing something related to this would be for symbionts that carry out different life stages in different hosts (things like Plasmodium). But I think that we'd want a different term for those cases, not this one. 
One question - I'm wondering if it makes sense to merge this into 'biological process involved in interaction with host' (new name)? rather than obsolete. Although if there's only one annotation, maybe it doesn't matter much.

**Comment by pgaudet** [comment]
I dont undertand how that term helps to annotate things like Plasmodium. Both the mosquito and the human phase of the life cycle would be annotated to the same term. 

If I interpret the image blow correctly, I think those annotations would be more informative: 
- 'GO:0000278 mitotic cell cycle' occurs in some Anopheles mosquito
- GO:0051321 meiotic cell cycle occurs in some Human



![image](https://user-images.githubusercontent.com/4782928/97262753-cef3be80-1821-11eb-8237-62afd94f14f0.png)

**Comment by pgaudet** [comment]
To your second point - @cmungall suggests we avoid merging when the term meaning is not the same (or not close enough). In this specific case (and related ones) we end up with synonyms that are really confusing, unless we know the history of how the synonyms got there, which I dont think most users are aware of, so I would lean towards obsoletion. 

Thanks, Pascale

**Comment by ValWood** [comment]
I also agree that for the GO term, that actual process plus location in host would be preferable.
For those annotating plasmodium in-depth, there is also a life cycle ontology? so 
occurs_during stage x as an extension (In the same way that we use phase terms as extensions in GO).

In the above case I would be inclined not to make a GO annotation beyond "interaction with host"    because we know nothing about the process. It is unlikely to be a cell-cycle process (especially as it refers to an effector  molecule), more likely some part of the invasion, migration etc. In fact, the final sentence of the abstract "This suggests that some Francisella pathogenicity island proteins were secreted while others remain within the bacterium during infection of host cells structural components of the secretion system and were necessary for secretion"
so basically these are  really likely to be required in some way for the secretion of other effectors.
I just don't think there is enough information here for anything more. At this stage it's no more than a reasonable speculation.

we would annotate the phenotype at PHI-base as 
decreased pathogen growth within host + decreased virulence

**Comment by mgiglio99** [comment]
I agree that obsoletion is the best thing to do.
